python/hail/methods/impex.py

```diff
@@ -1064,6 +1087,21 @@ def import_matrix_table(paths, row_fields={}, key=[], entry_type=tint32, missing
 
     Notes
     -----
+
+    The structure of the imported matrix table is:
+
+        - **row fields** are named as specified in the column header. If they
+        are missing from the header or ``no_header=True``, row field names are
+        set to the strings `f0`, `f1`, ... (0-indexed) in column order.
+
+        - **col_id** is the column key. It contains the IDs from the header of
```

Comment:
There is one column field **col_id**, which is a key field of type :obj:`str` or :obj:`int`. If ```no_header=False``, its values are the strings given by the corresponding column names in the header line. If  ``no_header=True``, its values are the integers 0, 1, ... (0-indexed) in column order.